src/tree_panel.py

```diff
@@ -1114,6 +1208,11 @@ def import_layers(self):
             if len(qgs_map_layers) == 0: # just in case if checkImportActionsAvailability() works incorrectly
                 return
 
+        items_to_upload_count = self.get_items_to_upload_count(qgs_map_layers, False)
```

Comment:
Combining it into a single method would make it cleaner (you did the same two-steps action at least twice)
```
if not self.subscription_limits_fit_layers(layers):
    self.display_update_subscription....
```